themes/admin/Sea_Green/css/sea_green.css

```
@@ -13335,7 +13335,7 @@ a.explorer-group:focus {
 }
 
 #token-grid table tr.odd .blank_button {
-  /*For older Browsers with compatibility issues*/
+  /*For older Browsers with compatibilty issues*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)